features/attached_enable.feature

```diff
@@ -93,13 +93,26 @@ Feature: Enable command behaviour when attached to an UA subscription
             ESM Infra is already enabled.
             See: sudo ua status
             """
+        When I run `apt-cache policy` with sudo
+        Then apt-cache policy for the following url has permission `500`
+        """
+        <esm-infra-url> <release>-infra-updates/main amd64 Packages
+        """
+        And I verify that running `apt update` `with sudo` exits `0`
+        When I run `apt install -y <infra-pkg>` with sudo
+        And I run `apt-cache policy <infra-pkg>` as non-root
+        Then stdout matches regexp:
+        """
+        \s*500 <esm-infra-url> <release>-infra-security/main amd64 Packages
+        \s*500 <esm-infra-url> <release>-infra-updates/main amd64 Packages
+        """
 
         Examples: ubuntu release
-           | release |
-           | bionic  |
-           | focal   |
-           | trusty  |
-           | xenial  |
+           | release | infra-pkg | esm-infra-url                       |
+           | bionic  | libkrad0  | https://esm.ubuntu.com/infra/ubuntu |
+           | focal   | hello     | https://esm.ubuntu.com/infra/ubuntu |
+           | trusty  | libgit2-0 | https://esm.ubuntu.com/ubuntu/      |
```

Comment:
I opened a pull request about this in ua-contracts. I *think* they should all be set the same, but we'll see what contracts team says 
https://github.com/CanonicalLtd/ua-contracts/issues/1115

Reply:
Great, if it lands, I will make another PR to remove that